source/_integrations/openuv.markdown

```diff
@@ -174,3 +174,21 @@ automation:
           - sensor.LATITUDE_LONGITUDE_current_uv_index
 ```
 {% endraw %}
+
+## Expired API Keys and Re-authentication
+
+In OpenUV, an `HTTP 403` response indicates one of two conditions:
+
+1. An invalid API key
+2. An API key whose daily/monthly limit is reached
+
+Unfortunately, the integration is unable to determine which is which from the API data
+provided by OpenUV. So, this strategy is followed:
+
+1. Any `HTTP 403` will trigger a re-auth flow.
+2. In the case of an overrun API call limit, once the limit expires and the coordinator
+   successfully retrieves data, any existing re-auth flow is canceled.
```

Comment:
Hmm, second thought. "re-auth flow" and "coordinator" are pretty much backend/developer terms. An average user won't have any idea what those mean. 

Can we generalize those to make it more clear what is going on?

Reply:
10-4. I'll think of something more user-friendly.

Reply:
Thoughts on https://github.com/home-assistant/home-assistant.io/pull/25671/commits/3a70d64b58d2357f1728be11b740298ac9e8a5dc?